STYLE_API_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
-# API reference page template
+# API reference style guide
 
-This template provides the basic structure for creating OpenSearch API documentation. It includes the most important elements that should appear in the documentation and helpful suggestions to help support them. 
+This style guide provides a template with the basic structure for OpenSearch API documentation. It includes the basic headings and helpful suggestions for what information is required in each section.
```

Comment:
This isn't a style guide, just a template.

Reply:
actually, a template would contain only pertinent sections with specific examples for each. We have provided a style guide with guidelines for how to write those sections.

Reply:
I think it's ok to have style guidelines in a template. Let's leave it.